dags/miovision_check.py

```diff
@@ -95,6 +98,40 @@ def miovision_check_dag():
     Identify gaps larger than gap_threshold in intersections with values today.
     '''
 
-    t_upstream_done >> [check_distinct_intersection_uid, check_gaps]
+    @task.short_circuit(ignore_downstream_trigger_rules=False, retries=0) #only skip immediately downstream task
+    def check_if_thursday(ds=None): #check if thursday to trigger weekly check. 
+        start_date = datetime.strptime(ds, '%Y-%m-%d')
+        return start_date.isoweekday() == 4
+    
+    check_open_anomalous_ranges = SQLCheckOperatorWithReturnValue(
+        task_id="check_open_anomalous_ranges",
+        sql="""WITH ars AS (
+            SELECT
+                uid,
+                intersection_name,
+                classification_uid,
+                notes
+            FROM gwolofs.open_issues
+            WHERE last_week_volume > 0
+            ORDER BY uid
+        )
+
+        SELECT
+            NOT(COUNT(*) > 0) AS _check,
+            CASE WHEN COUNT(*) = 1 THEN 'There is ' ELSE 'There are ' END || COUNT(*) ||
+                ' open ended anomalous_range(s) with non-zero volumes in the last 7 days: (uid, intersection_name, classification_uid, notes)'
```

Comment:
I think we can get rid of the stuff in the brackets

Reply:
Gonna leave, mostly to identify the cryptic uid (anomalous_range.uid) field.